packages/hooks/src/useAntdTable/index.tsx

```diff
@@ -162,7 +162,7 @@ const useAntdTable = <TData extends Data, TParams extends Params>(
       form.resetFields();
     }
     _submit({
-      ...(defaultParams?.[0] || {}),
+      ...{ ...params[0], ...defaultParams?.[0] },
```

Comment:
应该是 `...params[0]` 在后面吧？

```suggestion
      ...{ ...defaultParams?.[0], ...params[0] },
```

Reply:
这是重置，从最初的代码可以看出，是想重置为 defaultParams ，自然在改造后要置于后面覆盖 params[0] 中的其他参数

Reply:
这样做的好处是，不但可以重置 form ，还可以通过默认传参重置 sorter 和 filter